src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -4388,7 +4388,7 @@ namespace HeatBalanceSurfaceManager {
     // Beginning of Record Keeping subroutines for the HB Module
     // *****************************************************************************
 
-    void UpdateFinalSurfaceHeatBalance()
+    void UpdateFinalSurfaceHeatBalance(ChilledCeilingPanelSimpleData &dataChilledCeilingPanelSimple)
```

Comment:
I know long term we won't be able to keep this as just the chilled ceiling, and that's fine. I like that we are keeping it as low-level as we can.